packages/extension-link/src/helpers/clickHandler.ts

```diff
@@ -15,6 +15,12 @@ export function clickHandler(options: ClickHandlerOptions): Plugin {
           return false
         }
 
+        const eventTarget = event.target as HTMLElement
+
+        if (eventTarget.nodeName !== 'A') {
```

Comment:
This would still open the wrong link when I have my selection in Link A and click on Link B right?

Reply:
Good point. It seemed so to me also, but after checking out, when you have two links within the selection it will not open the wrong one. It always opens the one you clicked. 

In another case, if you have selected link A, and you click on the other link outside selection, this handler will not run.

In conclusion, it works in both cases.